server/lemmy_utils/src/settings.rs

```diff
@@ -115,12 +115,16 @@ impl Settings {
     format!("{}/api/v1", self.hostname)
   }
 
+  pub fn get_config_location() -> String {
+    env::var("LEMMY_CONFIG_LOCATION").unwrap_or("config/config.hjson".to_string())
```

Comment:
I would keep the variable CONFIG_FILE, instead of putting it here inline.

Reply:
Ah okay, Ya I can do that.